src/tests/datetime.rs

```diff
@@ -0,0 +1,21 @@
+use std::str::FromStr;
+
+#[test]
+fn rfc3339_to_datetime() {
+    let rfc = "2020-06-09T10:58:07.095Z";
+    let date = chrono::DateTime::<chrono::Utc>::from_str(rfc).unwrap();
+    assert_eq!(
+        crate::DateTime::parse_rfc3339_str(rfc).unwrap(),
+        crate::DateTime::from_chrono(date)
+    );
```

Comment:
can we add an assertion that converting the datetime back to a string matches the input?